connectivity/FEATURE_BLE/source/cordio/TESTS/cordio_hci/driver/CMakeLists.txt

```diff
@@ -3,10 +3,10 @@
 
 cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
 
-set(MBED_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../../../../../.. CACHE INTERNAL "")
+set(mbed-os_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../../../../../.. CACHE INTERNAL "")
 set(TEST_TARGET mbed-connectivity-ble-cordio-hci-driver)
 
-include(${MBED_PATH}/tools/cmake/mbed_greentea.cmake)
+include(${mbed-os_SOURCE_DIR}/tools/cmake/mbed_greentea.cmake)
```

Comment:
Sorry I missed to comment here, Do we need to change this as well because if it is top-level CMake similar to application one then PROJECT_NAME and CMAKE_PROJECT_NAME will be `mbed-connectivity-ble-cordio-hci-driver` and the `mbed-os_SOURCE_DIR` variable is not yet created

Reply:
The test should not care where it is.

How we create greentea shall be changed and this MBED_PATH should be fixed there. I'll create a separate issue for greentea tests, how they use MBED_PATH and get mbed-os in. I'll revert the changes here.

I'll keep this to fix the Mbed OS files (excluding greentea tests).

Reply:
Created https://github.com/ARMmbed/mbed-os/issues/14523 and I'll revert now greentea tests changes.